content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -0,0 +1,390 @@
+
+The *Memory Manager* employs hint generation protocol to yield the most suitable NUMA affinity for a pod. The *Memory Manager* feeds the central manager (*Topology Manager*) with these affinity hints. Based on both the hints and *Topology Manager* policy, the pod is rejected or admitted to the node.
+
+Moreover, *Memory Manager* ensures that the memory which a pod requests is allocated from a minimum number of NUMA nodes.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## How Memory Manager Operates?
+
+The Memory Manager currently offers the guaranteed memory (and hugepages) allocation for Pods in Guaranteed QoS class. To immediately put the Memory Manager into operation follow the guidelines in section [Memory Manager configuration](#memory-manager-configuration)), and subsequently, prepare and deploy a `Guaranteed` pod as illustrated in section [Placing a Pod in the Guaranteed QoS class](placing-a-pod-in-the-guaranteed-qos-class).
+
+The Memory Manager is a Hint Provider, and it provides topology hints for the Topology Manager which then aligns the requested resources according to these topology hints. It also enforces `cgroups` (i.e. `cpuset.mems`) for pods. The complete flow diagram concerning pod admission and deployment process is illustrated in [Memory Manager KEP: Design Overview][4].
+
+During this process, the Memory Manager updates its internal counters stored in [Node Map and Memory Maps][2] to manage guaranteed memory allocation. 
+
+The Memory Manager updates the Node Map during the startup and runtime as follows.
+
+### Startup
+
+This occurs once a node administrator employs `--reserved-memory` (section [Reserved memory flag](#reserved-memory-flag)). In this case, the Node Map becomes updated to reflect this reservation as illustrated in [Memory Manager KEP: Memory Maps at start-up (with examples)][5].
+
+The administrator must provide `--reserved-memory` flag when `static` policy is configured.
+
+### Runtime
+
+Reference [Memory Manager KEP: Memory Maps at runtime (with examples)][6] illustrates how a successful pod deployment affects the Node Map, and it also relates to how potential Out-of-Memory (OOM) situations are handled further by Kubernetes or operating system.
+
+Important topic in the context of Memory Manager operation is the management of NUMA groups. Each time pod's memory request is in excess of single NUMA node capacity, the Memory Manager attempts to create a group that comprises several NUMA nodes and features extend memory capacity. The problem has been solved as elaborated in [Memory Manager KEP: How to enable the guaranteed memory allocation over many NUMA nodes?][3]. Also, reference [Memory Manager KEP: Simulation - how the Memory Manager works? (by examples)][1] illustrates how the management of groups occurs. 

```

Comment:
We usually try to avoid referring to KEPs in documentation. They are OK to list for further reading but the page itself should provide examples and other details about the actual, shipped implementation.

Reply:
OK, I understand. In fact, we tried to keep the implementation consistent with the KEP.
If the implementation diverges from the KEP specification, it is reflected in this official documentation, e.g. the syntax for 
 `--reserved-memory=` differs between KEP and official doc.